### PR TITLE
Add  and Implement forgotPassword page

### PR DESCRIPTION
**Pull request by fisayomi**
#### What does this PR do?
Shows the Implementation of Forgot-password Page
Fix issue with delayed rendering of Components

#### Description of Task to be completed?
 A user should be able to change password if forgotten

#### How should this be manually tested?
- Sign up with a valid Email
- Click on Login Page to reset
- Enter Email.
- The mail should be available in your inbox

#### Any background context you want to provide?
Note that the content of the mail reflects `mealdock`. This will be changed in another PR

#### What are the relevant pivotal tracker stories?
[#166791294](https://www.pivotaltracker.com/story/show/166791294)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta
     charset="utf-8"
     name="viewport"
-    content="width=device-width,initial-scale=1">
+    content="minimum-scale=1, width=device-width,initial-scale=1, shrink-to-fit=no">
```
Comment (omorilewa) [inline public/index.html]: What does this help us do?

File: src/app/App.jsx [hunk 2]
```diff
@@ -18,17 +22,37 @@ const STYLES = (theme) => ({
   }
 });
 
+toast.configure();
+
 function App({ classes }) {
+  useEffect(() => {
+    document.body.style.height = '100%';
```
Comment (omorilewa) [inline src/app/App.jsx]: Why are we adding styles to the DOM this way?

File: public/index.html (hunk 1 above)
Comment (fisayomi) [reply public/index.html]: To make it responsive

File: src/app/App.jsx (hunk 2 above)
Comment (fisayomi) [reply src/app/App.jsx]: That's how it can be added for now to take care of the html and body outside the app.

**Comment by omorilewa** [comment]
I entered my email address and it says it is not a valid email
Screenshot 2019-06-21 at 6 08 29 PM

File: src/app/App.jsx (hunk 2 above)
Comment (omorilewa) [reply src/app/App.jsx]: I do not understand the `outside the app` bit. I noticed commenting these out does not allow a full page load. Was something else changed?

Comment (oreoluwade) [reply src/app/App.jsx]: @fisayomi, does "outside the app" mean outside the React App bounded by the `id-app` tag?

Comment (fisayomi) [reply src/app/App.jsx]: `Outside the app` is the entire wrapper, i.e the body and html in the html entry file.

**Pull request merged**